playground/backend/internal/preparers/go_preparators.go

```diff
@@ -13,33 +13,61 @@
 // See the License for the specific language governing permissions and
```

Comment:
Could you please also rename the file to `go_preparers.go`?

Reply:
Done